setup.cfg

```diff
@@ -5,24 +5,32 @@ description-file =
     README.rst
 author = Redfish dev team
 author-email = python-redfish@mondorescue.org
-home-page = http://github.com/bcornec/python-redfish
+home-page = https://github.com/uggla/python-redfish/tree/devel
```

Comment:
I think the best is to point to the root repo.
Same for issues.

I vote for adding ourselves and Vincent as contributors to the root repo.
- It will make management of issues easier. We will be able to own and qualify tickets.
- Shows project activity
- Correct contribution statistics.
- Flatten the project repo hierarchy.

But I'm in a favor of PR to commit code. Because it creates a kind of peer review.

Could you request project contributors right to Deva ?